wordpress-mobile/WordPress-Android
# UI modernization remove fab tool tip on posts screen

Removes FAB tooltip on Posts screen.  It was previously removed on Home.

Fixes: See comment  pcdRpT-3oR-p2#comment-5700

To test:

- Launch WP/JP app
- `Verify` FAB is present on home, and no tool tip

- Navigate to the Posts screen (Menu -> Posts)
- Navigate to Published, Drafts, Scheduled, and Trashed tabs
- `Verify` FAB is present, but no tool tip on it

- Navigate to the Pages screen (Menu -> Pages)
- Navigate to Published, Drafts, Scheduled, and Trashed tabs
- `Verify` FAB is present, but no tool tip on it

- For completeness, try the above steps with the JP app as well.

## Regression Notes
1. Potential unintended areas of impact
None

2. What I did to test those areas of impact (or what existing automated tests I relied on)
Tested manually

3. What automated tests I added (or what prevented me from doing so)
Updated existing unit tests

PR submission checklist:


UI Changes testing checklist:
